repose-aggregator/commons/commons-configuration/src/main/java/org/openrepose/commons/config/parser/jaxb/JaxbConfigurationParser.java

```diff
@@ -48,10 +48,6 @@ public JaxbConfigurationParser(Class<T> configurationClass, JAXBContext jaxbCont
         super(configurationClass);
         if (xsdStreamSource == null) {
             LOG.warn("Creating a JAXB Parser Pool without any schema to validate for {}", configurationClass);
-            if (LOG.isDebugEnabled()) {
-                Exception tracer = new Exception("Repose Devs might care about this trace");
-                LOG.debug("Logging the current stack to find where a parser pool is created without a validator", tracer);
-            }
```

Comment:
We really should not care about this. There are plenty of cases where we listen to a configuration without validating it; after all, a configuration really only needs to be validated once (as long as it is done in the right place). Additionally, not validating a configuration is an explicit choice by the developer. This log message seemed to be added to help track down cases where we may have wanted to validate the configuration but did not do so. Presumably those cases were addressed. In other words, this log message seemed to be a debugging tool for a developer, not the user. In the end, the user can't change anything, so why show them this message? As for the developer, well, we get the `warn` message from above which should be enough information to fix any issues.